week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,15 @@
 */
 
 function calculateGrade(mark) {
-
+  if(mark >= 80){
+    return 'A';
+  }else if(60 <= mark && mark < 80){
```

Comment:
You don't actually need `mark < 80` here because we have already evaluated that in line 12, and got to the "else" clause of it.